lib/quizzy.rb

```diff
@@ -56,3 +57,8 @@
     # production :stmp, address: ENV['SMTP_PORT'], port: 1025
   end
 end.load!
+
+
+lib = File.expand_path('..', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.